torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -78,6 +78,16 @@ def _get_default_op_backend_config(op, dtype_configs):
         torch.nn.functional.layer_norm,
     ]]
 
+_ADD_CONFIG = {
+    "pattern": operator.add,
+    "observation_type": ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT,
```

Comment:
looks like this is not always true, can we also put the other observation type in this config instead of hiding it?

Reply:
It could be something like

```
{
  "observation_type": ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT,
  "observation_type_if_binary_op_and_one_input_scalar": ObservationType.OUTPUT_SHARE_OBSERVER_WITH_INPUT,
}
```

or

```
{
  "observation_type": ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT_EXCEPT_ONE_INPUT_SCALAR_THEN_SHARE_OBSERVER_WITH_INPUT,
}
```

we should also make sure we have a way to change observation type of scalar add in the future if we need to

Reply:
maybe I can have a map from num_tensor_args to the observation type?